virtool/references/utils.py

```diff
@@ -266,12 +267,13 @@ def get_otu_schema(require_id: bool) -> dict:
     }
 
 
-def get_owner_user(user_id: str) -> dict:
+def get_owner_user(user_id: str, created_at: Optional[datetime]) -> dict:
```

Comment:
`create_document` in references/db calls `get_owner_user`, but `created_at` is a keyword argument set to None. Therefore I just typed it with Optional and assigned it to `virtool.utils.timestamp` in case it was None.

Reply:
I've reviewed this and it looks like `create_document()` is always passed a value for `created_at`. It should not be optional or have a default value of `None`.

Can you verify this and make the change accordingly.